jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/IPC.java

```diff
@@ -85,8 +88,16 @@ public class IPC {
 	 */
 	public static final boolean isZOS;
 
+	/**
+	 * True - using FileLockWatchdogTask when operation system is not z/OS, or
```

Comment:
This seems strangely worded. How about something like: Controls use of the FileLockWatchdogTask. Always set true on z/OS, defaults to false otherwise. Setting -Dcom.ibm.tools.attach.disableFileLockWatchdog=true overrides the default value on non-z/OS platforms.

Reply:
Updated